pkg/apiserver/config/config_test.go

```diff
@@ -179,6 +180,10 @@ func newTestConfig() (*Config, error) {
 		MeteringOptions: &metering.Options{
 			RetentionDay: "7d",
 		},
+		GatewayOptions: &gateway.Options{
+			WatchesPath: "/etc/kubesphere/watches.yaml",
```

Comment:
What does the `watches.yaml` look like, and why do we need it?

Reply:
Please check the comments in the PR (2nd item). Generally, it's a mapping relation between CRD and the Helm Chart. The operator will monitor the CRD configured in the file, when CR is created and a corresponding Helm Release will be created too.
Currently, the Helm Operator SDK only supports file path-based parameters. We may replace it with CRD in the feature.